### ISSUE TITLE
Redundant constraints in `storage_symmetric`

### ISSUE THREAD

**Issue by cfe316** [GenXProject/GenX#566]
```julia
# Maximum charging rate (including virtual charging to move energy held in reserve back to available storage) must be less than symmetric power rating
[y in STOR_SYMMETRIC, t in 1:T], EP[:vCHARGE][y,t] + EP[:vCAPRES_charge][y,t] <= EP[:eTotalCap][y]

# Max simultaneous charge and discharge cannot be greater than capacity
[y in STOR_SYMMETRIC, t in 1:T], EP[:vP][y,t]+EP[:vCHARGE][y,t]+EP[:vCAPRES_discharge][y,t]+EP[:vCAPRES_charge][y,t] <= EP[:eTotalCap][y]
```
* all these quantities are positive
* the LHS of the second constraint equals the LHS of the first constraint plus some extra terms.
* both RHS's are the same.
Therefore the first constraint is redundant. Not sure if we want to remove it, or keep it for future expansion/didactic reasons.

**Comment by cfe316** [comment]
This would be addressed by #562 .

**Comment by cfe316** [comment]
The fix for this has been merged to develop.
